ViewModel/ImpactAreaScenario/Editor/SpecificIASEditorVM.cs

```diff
@@ -417,49 +417,59 @@ private List<StageDamageCurve> GetStageDamageCurves()
 
         #region validation
 
-        private string IsFrequencyRelationshipValid()
+        private FdaValidationResult IsFrequencyRelationshipValid()
         {
-            return SelectedFrequencyElement.ChildElement == null ? "A Frequency Relationship is required." : null;      
+            FdaValidationResult vr = new FdaValidationResult();
+            if(SelectedFrequencyElement.ChildElement == null)
+            {
+                vr.AddErrorMessage("A Frequency Relationship is required.");
+            }
+            return vr;
         }
 
-        private string IsRatingCurveValid()
+        private FdaValidationResult IsRatingCurveValid()
         {
             //todo: the rating curve is required if the frequency relationship is of type
             //flow-frequency. This will need to get added once we complete task 5 in the clean doc.
-            string msg = null;
+            FdaValidationResult vr = new FdaValidationResult();
             if (_ratingRequired && SelectedRatingCurveElement.ChildElement == null)
             {
-                msg = "A Rating Curve is required when using a frequency relationship.";
+                vr.AddErrorMessage("A Rating Curve is required when using a frequency relationship.");
             }
-            return msg;
+            return vr;
+
         }
-        private string IsStageDamageValid()
+        private FdaValidationResult IsStageDamageValid()
```

Comment:
Follow the same naming conventions as GetPlotValidationResults.  Is implies bools are returned.